uint8 LINE = 0
uint8 CIRCLE = 1
uint8 TRIANGLE = 2

uint8 type
float64 circle_yc
float64 circle_zc
float64 radius
float64 triangle_yt
float64 triangle_zt
float64 tiangle_side_length